setup.py

```diff
@@ -0,0 +1,34 @@
+import distutils.core
+from distutils.command.install import install
+import subprocess
+
+
+class PyenvInstall(install):
+    def run(self):
+        print(subprocess.check_output(['bash', 'bin/pyenv-installer']))
+
+with open('README.rst') as file:
+    long_description = file.read()
+
+distutils.core.setup(
+    version='20150113',
```

Comment:
The `pyenv` on PyPI is actually a `pyenv-installer`. The `pyenv` and `pyenv-installer` can be versioned individually. Basically, I think we don't have to release the pyenv and pyenv-installer together. It might be somewhat confusing, but I think it's acceptable.

We all human need more improvement on our toolchain... Though, current impl is useful enough to start from here. Keep on hacking :)